alws/crud.py

```diff
@@ -31,6 +31,11 @@ async def create_build(
         await planner.load_platforms()
         for task in build.tasks:
             await planner.add_task(task)
+        if build.linked_builds:
```

Comment:
After adding pydantic.Field, this line can be erased

Reply:
nope